.gitlab-ci.yml

```diff
@@ -126,7 +126,8 @@ rubocop:
 debian-debian-10-master-py3: {extends: '.test_instance'}
 # ubuntu-ubuntu-2004-master-py3: {extends: '.test_instance'}
 ubuntu-ubuntu-1804-master-py3: {extends: '.test_instance'}
-# centos-centos-8-master-py3: {extends: '.test_instance'}
+redhat8-centos-8-master-py3: {extends: '.test_instance'}
+redhat8-oraclelinux-8-master-py3: {extends: '.test_instance'}
```

Comment:
This line will end up getting moved lower down in the file once I update the `ssf-formula` with this PR but nothing to worry about (I'll just push the commit on top after this PR is merged).  Just for your info, @aboe76.